lib/rubygems/resolver/installer_set.rb

```diff
@@ -193,7 +193,7 @@ def load_spec name, ver, platform, source # :nodoc:
   # Has a local gem for +dep_name+ been added to this set?
 
   def local? dep_name # :nodoc:
-    spec, = @local[dep_name]
+    spec = @local[dep_name]
```

Comment:
Hey, thanks for contributing to rubygems :)

I don't think that comma is just for style purposes, that comma there ensure we get the spec part from `@local[dep_name]`
since `@local[dep_name]` can be equal to `[spec, source]`